src/Sentry.Unity.Editor/AutoInstrumentation/SentryAssemblyResolver.cs

```diff
@@ -0,0 +1,40 @@
+using System.IO;
+using Mono.Cecil;
+
+namespace Sentry.Unity.Editor
+{
+    public class SentryAssemblyResolver : BaseAssemblyResolver
+    {
+        private readonly string _workingDirectory;
+        private readonly DefaultAssemblyResolver _defaultResolver;
+
+        public SentryAssemblyResolver(string workingDirectory)
+        {
+            _workingDirectory = workingDirectory;
+            _defaultResolver = new DefaultAssemblyResolver();
+        }
+
+        public override AssemblyDefinition Resolve(AssemblyNameReference name)
+        {
+            AssemblyDefinition assembly;
+            try
+            {
+                assembly = _defaultResolver.Resolve(name);
+            }
+            catch (AssemblyResolutionException e)
+            {
+                var path = Path.GetFullPath(Path.Combine(_workingDirectory, e.AssemblyReference.Name + ".dll"));
```

Comment:
Is `Path.GetFullPath` necessary?  I think both `File.Exists` and `AssemblyDefinition.ReadAssembly` would both resolve relative paths the same way.

Reply:
Apparently not. I did a double-check and I get around the whole `SentryResolver` by adding to the search directories of a `DefaultAssemblyResolver`! 👍 
Thanks for pointing that out.